packages/eslint-plugin-react-hooks/src/ExhaustiveDeps.js

```diff
@@ -93,6 +93,28 @@ export default {
       const reactiveHook = node.callee;
       const reactiveHookName = getNodeWithoutReactNamespace(reactiveHook).name;
       const declaredDependenciesNode = node.arguments[callbackIndex + 1];
+      const isEffect = /Effect($|[^a-z])/g.test(reactiveHookName);
```

Comment:
Would "Effects" be a valid name for this? `useSomeCustomEffects(fn, deps)`

Reply:
Not with the current check, but you can use `additionalHooks` for that.